core/src/main/res/values/strings.xml

```diff
@@ -136,12 +136,12 @@
     <string name="feed_settings_label">Podcast settings</string>
     <string name="rename_feed_label">Rename podcast</string>
     <string name="remove_feed_label">Remove podcast</string>
-    <string name="share_label">Share&#8230;</string>
+    <string name="share_label">Share</string>
```

Comment:
I think it would be better to have 2 strings here. One with the dots and one without. Menu items with sub menus usually have dots in their title but the dialog title and button are better without dots. Let's wait for a reply by @keunes first. @keunes, what do you think?

Reply:
Yup, agreed.

Reply:
Yup, I like the idea too